.github/workflows/maven.yml

```diff
@@ -232,13 +232,18 @@ jobs:
         uses: aquasecurity/trivy-action@master
         env:
           TRIVY_DOWNLOAD_JAVA_DB_ONLY: true
+          TRIVY_JAVA_DB_REPOSITORY: public.ecr.aws/aquasecurity/trivy-java-db,ghcr.io/aquasecurity/trivy-java-db:1
         with:
           scan-type: image
+          timeout: 10m0s
           cache-dir: .trivy
 
       - name: Trivy Vulnerability Scan
         if: ${{ matrix.os == 'ubuntu-latest' }}
         uses: aquasecurity/trivy-action@master
+        env:
+          TRIVY_SKIP_DB_UPDATE: true
```

Comment:
Note we haven't actually download the main DB at this point, only the Java one

However I think this is still fine because the Trivy Cache Action won't have restored the cache if the most recent DB SHA has changed which will force a first time attempt to download the DB

We could add an explicit download main DB step similar to the Download Java DB step if we wanted to make it clear when builds are failing due to failure to download the DB vs scan failure.  Basically copy/paste the Download Java DB step and remove `_JAVA_` and `-java-` from the various bits

Reply:
Given the repeated continual failures, it suggests trying to download the DB during the action. 

I'll make the additional changes